teamcity.sh

```diff
@@ -11,6 +11,8 @@ npm install -g yarn
 
 yarn install
 
+yarn lint
+
```

Comment:
This is unrelated, but I think if we have linting, we probably want to keep `main` error free